docs/vendors/index.md

```diff
@@ -0,0 +1,23 @@
+
+
+## Vendor Overview
+
+### Comparison
+
+|              | Heroku        | Digital Ocean  | Azure         | T1Pal        |
+| ------------ | ------------- | -------------- | ------------- | ------------ |
+| Install Day 1| DIY           | DIY            | DIY           | Managed      |
+| Maintenance Day 2 | DIY       | DIY            | DIY           | Automatic    |
+| Cost         | Free, sales emails | $5USD/month | Depends       | $11.99USD/month |
+| Support      | DIY / Facebook| DIY / Facebook | DIY / Facebook | Private support |
+| Community Contributions |  None | None        | Research use | contributors  |
```

Comment:
Ran this through [table-magic](https://stevecat.net/table-magic) to get even columns:
```suggestion
|                         | Heroku             | Digital Ocean  | Azure          | T1Pal              |
|-------------------------|--------------------|----------------|----------------|--------------------|
| Install Day 1           | DIY                | DIY            | DIY            | Managed            |
| Maintenance Day 2       | DIY                | DIY            | DIY            | Automatic          |
| Cost                    | Free, sales emails | $5/month (USD) | Depends        | $11.99/month (USD) |
| Support                 | DIY / Facebook     | DIY / Facebook | DIY / Facebook | Private support    |
| Community Contributions | None               | None           | Research use   | contributors       |
|                         |                    |                |                |                    |
```

Reply:
Also a thought, might want to invert the columns and rows if the providers will be added to, but the criteria will stay static.

Reply:
@bewest good idea, but how do we handle MongoDB Atlas that's paired with Heroku in classic DIY, and can be used with Azure, but isn't involved with DO and others... Maybe an extra line for Database?

Reply:
I think the criteria is not quite a fixed set, yet.  This table only reflects web-hosting, not database hosting.  Nightscout requires both web host and database host.  For example, DigitalOcean, people can make a choice: do they want built-in mongo or an external mongo.  We will likely document both options.  Should we add a criteria for database: self-contained vs external provider vs optional?

Reply:
>  Should we add a criteria for database: self-contained vs external provider vs optional?

I think yes.

Reply:
`Components, costs and complexity will depend on the solution you want to implement.

Some solutions below:



|                              | Heroku              | Digital Ocean           | MVPS                    | Azure               | T1Pal             |
| ---------------------------- | ------------------- | ----------------------- | ----------------------- | ------------------- | ----------------- |
| Install                      | DIY                 | DIY                     | DIY                     | DIY                 | Managed           |
| Maintenance                  | DIY                 | DIY                     | DIY                     | DIY                 | Automatic         |
| Cost (USD)                   | Free                | $5/month                | $3/month                | Depends             | $11.99/month      |
| Support                      | DIY / Facebook      | DIY / Facebook          | DIY / Facebook          | DIY / Facebook      | Private support   |
| Community Contributions      | None                | None                    | None                    | Research use        | Contributors      |
| Mongo database<br />External | None<br />Mandatory | Contained<br />Optional | Contained<br />Optional | None<br />Mandatory | Contained<br />No |
| Complexity                   | Medium              | High                    | High                    | High                | Low               |`